File: QViz.Java/src/main/java/com/periscope/qviz/json/TestCaseAPI.java
package com.periscope.qviz.json;

import java.util.ArrayList;
import java.util.List;

/**
 * JSON Model for API Test Case Details
 */
public class TestCaseAPI extends TestCase {

	/**
	 * List of Test Case Step Objects associated with the Test Case
	 */
	public List<TestCaseStep> testCaseSteps;

	/**
	 * Lise of Test Action Objects associated with the Test Case
	 */
	public List<Action> testActions;

	/**
	 * List of Test API Objects associated with the Test Case
	 */
	public List<TestAPI> testAPIs;

	/**
	 * Default Constructor
	 */
	public TestCaseAPI() {
		super();
		this.testCaseSteps = new ArrayList<>();
		this.testActions = new ArrayList<>();
		this.testAPIs = new ArrayList<>();
	}

}

File: QViz.NET/Periscope.QViz/JSON/TestCaseAPI.cs
using System.Collections.Generic;

namespace Periscope.QViz.JSON
{
	/// <summary>
	/// JSON Model for API Test Case Details
	/// </summary>
	public class TestCaseAPI : TestCase
	{
		/// <summary>
		/// List of Test Case Step Objects associated with the Test Case
		/// </summary>
		public IList<TestCaseStep> testCaseSteps;
		
		/// <summary>
		/// Lise of Test Action Objects associated with the Test Case
		/// </summary>
		public IList<Action> testActions;
		
		/// <summary>
		/// List of Test API Objects associated with the Test Case
		/// </summary>
		public IList<TestAPI> testAPI;

		/// <summary>
		/// Default Constructor
		/// </summary>
		public TestCaseAPI() : base()
		{
			this.testCaseSteps = new List<TestCaseStep>();
			this.testActions = new List<Action>();
			this.testAPI = new List<TestAPI>();
		}
	}
}